README.md

```diff
@@ -1,47 +1,134 @@
-# Drupal Commerce Documentation
```

Comment:
🤦‍♂ forgot this overrides our custom README.

Reply:
That happened last time, too. We can just merge this and then I can open a second PR to restore the original/custom README.